README.md

```diff
@@ -2,40 +2,16 @@
 An toolkit for autonomous services implementing a decentralized Impact Evaluator built with the [Open Autonomy](https://docs.autonolas.network/open-autonomy/) framework.
 
 The demo service for the IEKit tracks mentions of [@autonolas](https://twitter.com/autonolas) on Twitter, assigns scores to them and writes those scores to a [Ceramic](https://ceramic.network/) stream.
-The service then:
 
-1. reads verified wallets (a mapping between Twitter handles and Ethereum addresses) from another Ceramic stream,
-2. checks for users that have minted a token, and
-3. updates their token metadata according to their scores so NFT images are upgraded over time.
+## Running a demo the service
```

Comment:
```suggestion
## Running a demo service
```

Reply:
There was a spelling issue so I included this change myself directly